src/FixedDataTable.js

```diff
@@ -1171,7 +1249,14 @@ class FixedDataTable extends React.Component {
 
     // only call onScrollStart if scrolling wasn't on previously
     if (!oldScrolling && scrolling && onScrollStart) {
-      onScrollStart(oldScrollX, oldScrollY, oldFirstRowIndex, oldEndRowIndex);
```

Comment:
For consistency with both `onScrollEnd` and `onViewportChange`, I'm also reporting the first/last column indexes.